pkg/util/util_test.go

```diff
@@ -27,9 +27,15 @@ func TestSetupFlyteDir(t *testing.T) {
 	assert.Nil(t, SetupFlyteDir())
 }
 
-func TestPrintSandboxMessage(t *testing.T) {
+func TestPrintSandboxStartMessage(t *testing.T) {
 	t.Run("Print Sandbox Message", func(t *testing.T) {
-		PrintSandboxMessage(SandBoxConsolePort, docker.SandboxKubeconfig, false)
+		PrintSandboxStartMessage(SandBoxConsolePort, docker.SandboxKubeconfig, false)
+	})
+}
+
+func TestPrintSandboxTeardownMessage(t *testing.T) {
```

Comment:
This test doesn't seem to be very useful, so just for the completeness.

Reply:
in all honesty, the previous test is not terribly useful either. It's fine to leave both there.